files/provision-user

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env bash
 set -euo pipefail
+
 if [ ! -f "/etc/kdk/provisioned" ]; then
-    useradd ${KDK_USERNAME} -m -G wheel -s "${KDK_SHELL}" >> /var/log/kdk-provision.log 2>&1
+    useradd ${KDK_USERNAME} -m -G wheel -s "${KDK_SHELL}"
     chown ${KDK_USERNAME} /home/${KDK_USERNAME}
-    runuser -l ${KDK_USERNAME} -c "yadm clone --bootstrap ${KDK_DOTFILES_REPO}" >> /var/log/kdk-provision.log 2>&1
+    touch /var/log/kdk-provision.log && chown ${KDK_USERNAME} /var/log/kdk-provision.log && chmod 0660 /var/log/kdk-provision.log
```

Comment:
May: 0600 would be safer here, because root can already r/w a file of 0600 owned by $USER.  No other user needs to r/w beyond root, so I would go with 0600.